Add piecewise tone mapping option

VictorNicolasLO opened the pull request:
Hey! I think this new new options could be useful, specially for those that have displays with aggresive abl, it reescalates the tone mapping and increase or decrease the brightness, so we can have an hdr 700 in hdr 1000 mode and gain some brightness, also I included another  option that does the same but only adjust the curve to modify the brightness without changing the clipping values
![image](https://github.com/user-attachments/assets/0e27b018-f84b-46d1-b396-20c878900a50)

VictorNicolasLO commented:
Hey @Maassoft! please take a look if you have a chance and let me know what you think

Maassoft commented:
Thanks for these new options! I will merge these with the master branch as soon as I find the time. Also the UI changes need to be done to the new UI (Blazor) because eventually the WinForms UI will be deprecated.

VictorNicolasLO commented:
Hi @Maassoft! I changed the "curve like" setting by "Brightness multiplier" and added a new "gamma multiplier" too also including the new feature to the new UI
![image](https://github.com/user-attachments/assets/41b0d29d-ec14-4e5d-a103-972fa22b3c93)
![image](https://github.com/user-attachments/assets/7e1be143-b827-40cf-b5ca-e37a260cead5)
